<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>QmitkXnatExperimentWidgetControls</class>
  <widget class="QWidget" name="QmitkXnatExperimentWidgetControls">
    <property name="enabled">
      <bool>true</bool>
    </property>
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>225</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Qmitk Template</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="0">
        <widget class="QLabel" name="timeLabel">
          <property name="text">
            <string>Time:</string>
          </property>
        </widget>
      </item>
      <item row="5" column="0">
        <widget class="QLabel" name="dateLabel">
          <property name="text">
            <string>Date:</string>
          </property>
        </widget>
      </item>
      <item row="7" column="0">
        <widget class="QLabel" name="scannerLabel">
          <property name="text">
            <string>Scanner:</string>
          </property>
        </widget>
      </item>
      <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="columnLabel">
          <property name="font">
            <font>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
            </font>
          </property>
          <property name="text">
            <string>Experiment Information</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
        </widget>
      </item>
      <item row="8" column="0">
        <widget class="QLabel" name="modalityLabel">
          <property name="text">
            <string>Modality:</string>
          </property>
        </widget>
      </item>
      <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="breadcrumbLabel">
          <property name="font">
            <font>
              <italic>true</italic>
            </font>
          </property>
          <property name="text">
            <string />
          </property>
        </widget>
      </item>
      <item row="3" column="0">
        <widget class="QLabel" name="labelLabel">
          <property name="text">
            <string>ID*:</string>
          </property>
        </widget>
      </item>
      <item row="9" column="0" colspan="2">
        <widget class="QLabel" name="mandatoryLabel">
          <property name="text">
            <string>* mandatory field</string>
          </property>
        </widget>
      </item>
      <item row="3" column="1">
        <widget class="QLineEdit" name="labelLineEdit">
          <property name="readOnly">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="5" column="1">
        <widget class="QLineEdit" name="dateLineEdit">
          <property name="readOnly">
            <bool>true</bool>
          </property>
          <property name="placeholderText">
            <string>YYYY-MM-DD</string>
          </property>
        </widget>
      </item>
      <item row="6" column="1">
        <widget class="QLineEdit" name="timeLineEdit">
          <property name="readOnly">
            <bool>true</bool>
          </property>
          <property name="placeholderText">
            <string>hh:mm:ss</string>
          </property>
        </widget>
      </item>
      <item row="7" column="1">
        <widget class="QLineEdit" name="scannerLineEdit">
          <property name="readOnly">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="8" column="1">
        <widget class="QLineEdit" name="modalityLineEdit">
          <property name="readOnly">
            <bool>true</bool>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources />
  <connections />
</ui>
